bot.py

```diff
@@ -29,12 +29,23 @@ def handle_image(message):
 
     # Construct the URL for downloading the photo
     photo_url = f"https://api.telegram.org/file/bot{TELEGRAM_TOKEN}/{file_info.file_path}"
-    print("URL:", photo_url)
 
     filename = str(message.chat.username) + "_" + str(message.chat.id) + "_" + str(message.id) + ".mp3"
 
     response = requests.get("http://localhost:8000/process_image/", json={'filename': filename, 'photo_url': photo_url})
 
+    # some information for debug
+    img_crp = response.json().get('cropped_img')
+    kn_distance = response.json().get('distance')
+    metadata = response.json().get('metadata')
+
+    bot.send_message(message.chat.id, f'DB: Distance {kn_distance}')
+    bot.send_message(message.chat.id, f'DB: Metadata')
+    bot.send_message(message.chat.id, metadata)
+
+    with open(img_crp, 'rb') as photo_crp:
+        bot.send_photo(message.chat.id, photo=photo_crp, caption="DB: Cropped image")
```

Comment:
Can we make this logic switchable? Like `if DEBUG: ...`, and add `DEBUG` to `.env` and `.env.example` files?

Reply:
done